dotnet-roslyn/config/Audacia.CodeAnalysis.AspNetCore/Audacia.CodeAnalysis.AspNetCore.csproj

```diff
@@ -15,7 +15,7 @@
     </ItemGroup>
 
     <ItemGroup>
-        <PackageReference Include="AspNetCoreAnalyzers" Version="0.2.0.1-dev" />
+        <PackageReference Include="AspNetCoreAnalyzers" Version="0.3.0" />
```

Comment:
Are there rules to review since upgrading this?

Reply:
Reverted as these are upgrading quite a few versions and not in the scope of this work